### PR TITLE
Add global config

### PR DESCRIPTION
**Pull request by nuclearcat**
Implement support for a global configuration file to enable kci-dev functionality across any source directory location, but keep support of local configuration file. This enhancement will allow for more flexible project setup and improved workflow portability.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ poetry run kci-dev
 
 ## Configuration
 
-kci-dev uses a configuration file .kci-dev.toml in the program directory.
+kci-dev uses a configuration file .kci-dev.toml in the current program directory
+or global configuration directory, in user's home directory/.config/kci-dev.toml.
```
[the remark this answers is not in the document]
Comment (aliceinwire) [reply docs/index.md:20]: global configurations are under /etc/ in most OS

[the remark this answers is not in the document]
Comment (aliceinwire) [reply docs/index.md:20]: current program directory  configuration ".kci-dev.toml" is per SITE configuration

[the remark this answers is not in the document]
Comment (aliceinwire) [reply docs/index.md:20]: and loading order is generally

- Global
- User
- Site

[the remark this answers is not in the document]
Comment (nuclearcat) [reply docs/index.md:20]: Ok, i will update. About loading order, do you mean "site" for example override previous ones or global have highest priority and will be used as highest priority (if available)?

[the remark this answers is not in the document]
Comment (aliceinwire) [reply docs/index.md:20]: global have highest priority, if present

[the remark this answers is not in the document]
Comment (aliceinwire) [reply docs/index.md:20]: Each file read overrides any values read from previous files

site for example override previous ones

[the remark this answers is not in the document]
Comment (aliceinwire) [reply docs/index.md:20]: i also added the kci-dev settings options in the list

**Comment by aliceinwire** [comment]
any update on this?

**Pull request merged**